### DISCUSSION TITLE
0.0.11

### DISCUSSION THREAD

**Discussion by mgmeyers** [mgmeyers/obsidian-kanban#32] [Releases]
- Refactor menu system
- Add "New note from card" menu option. This will create a new note with the card's contents and link the note to the card.

<img width="459" alt="Screen Shot 2021-04-25 at 9 37 05 PM" src="https://user-images.githubusercontent.com/2694747/116029526-689ffb00-a60e-11eb-9fc5-e0132dcb9e67.png">

<hr /><em>This discussion was created from the release <a href='https://github.com/mgmeyers/obsidian-kanban/releases/tag/0.0.11'>0.0.11</a>.</em>

**Comment by phibr0** [comment]
I am missing the quick archiving button for items marked "done".. The Option is still there in the new Menu but it is a bit tidious if you have multiple cards

**Reply by mgmeyers** [reply]
Agreed. I'll add it back shortly

**Comment by tf13** [comment]
Will there be an option to have links from cards to other notes (or ideally, elsewhere in the file system)?

**Reply by mgmeyers** [reply]
Could you elaborate @tf13? Not sure exactly what you mean. Currently you can manually link a card to any note:

<img width="452" alt="Screen Shot 2021-04-26 at 1 51 07 PM" src="https://user-images.githubusercontent.com/2694747/116149133-9e88c200-a696-11eb-9b8e-66f0e218e7fe.png">

**Reply by tf13** [reply]
OK, I missed that. For some reason the double-`[[` doesn't trigger a dropdown menu for me, but I suspect I turned something off in Obsidian settings that I've forgotten about. I did try to wiki link that way to a few notes, but clicking on the resulting link created a note every time, whether or not it existed. It's very possible this is user error — I'm pretty new to Obsidian and don't ordinarily use wikilinks.

What I'd really like is to be able to link to a directory on my Mac. I want to use the Kanban plugin to track projects, and each project has a folder in the file system (usually not in the Obsidian vault). I'd love to be able to click to open that folder...
